src/api.js

```diff
@@ -10,6 +10,26 @@ export function getArticles() {
   });
 }
 
+export function getSortedArticles(sort, order) {
+  let endpoint = `/api/articles?`;
+
+  endpoint += `sort_by=${sort}&order=${order}`;
+
+  return api.get(endpoint).then(({ data: { articles } }) => {
+    return articles;
+  });
+}
+
+export function getSortedTopicArticles(topic, sort, order) {
+  let endpoint = `/api/articles?`;
+
+  endpoint += `topic=${topic}&sort_by=${sort}&order=${order}`;
+
+  return api.get(endpoint).then(({ data: { articles } }) => {
+    return articles;
+  });
+}
```

Comment:
An easier way to do this is with axios query object
https://masteringjs.io/tutorials/axios/get-query-params